From 203e3cb2901f6d3b8bce393e8e3c7b72ce981b33 Mon Sep 17 00:00:00 2001
From: Kalle Jokiniemi <kalle.jokiniemi@jolla.com>
Date: Fri, 29 Aug 2014 10:42:22 +0300
Subject: [PATCH 2/2] [doc] remove documentation building

Mer is missing asciidoc package needed by btrfs-progs documentation build.
Removing the documentation build from the Makefile to avoid the
dependency and to make builds work.

Signed-off-by: Kalle Jokiniemi <kalle.jokiniemi@jolla.com>
---
 Makefile | 13 +++----------
 1 file changed, 3 insertions(+), 10 deletions(-)

diff --git a/Makefile b/Makefile
index f8b928c..88f5191 100644
--- a/Makefile
+++ b/Makefile
@@ -57,8 +57,8 @@ btrfs_image_libs = -lpthread
 btrfs_fragments_libs = -lgd -lpng -ljpeg -lfreetype
 
 SUBDIRS =
-BUILDDIRS = $(patsubst %,build-%,$(SUBDIRS)) build-Documentation
-INSTALLDIRS = $(patsubst %,install-%,$(SUBDIRS)) install-Documentation
+BUILDDIRS = $(patsubst %,build-%,$(SUBDIRS))
+INSTALLDIRS = $(patsubst %,install-%,$(SUBDIRS))
 CLEANDIRS = $(patsubst %,clean-%,$(SUBDIRS))
 
 .PHONY: $(SUBDIRS)
@@ -220,10 +220,7 @@ send-test: $(objects) $(libs) send-test.o
 	@echo "    [LD]     $@"
 	$(Q)$(CC) $(CFLAGS) -o send-test $(objects) send-test.o $(LDFLAGS) $(LIBS) -lpthread
 
-manpages:
-	$(Q)$(MAKE) $(MAKEOPTS) -C Documentation
-
-clean-all: clean-doc clean
+clean-all: clean
 
 clean: $(CLEANDIRS)
 	@echo "Cleaning"
@@ -233,9 +230,6 @@ clean: $(CLEANDIRS)
 	      version.h $(check_defs) \
 	      $(libs) $(lib_links)
 
-clean-doc:
-	@echo "Cleaning Documentation"
-	$(Q)$(MAKE) $(MAKEOPTS) -C Documentation clean
 
 $(CLEANDIRS):
 	@echo "Cleaning $(patsubst clean-%,%,$@)"
@@ -258,7 +252,6 @@ $(INSTALLDIRS):
 	$(Q)$(MAKE) $(MAKEOPTS) -C $(patsubst install-%,%,$@) install
 
 uninstall:
-	$(Q)$(MAKE) $(MAKEOPTS) -C Documentation uninstall
 	cd $(DESTDIR)$(incdir); rm -f $(headers)
 	rmdir -p --ignore-fail-on-non-empty $(DESTDIR)$(incdir)
 	cd $(DESTDIR)$(libdir); rm -f $(lib_links) $(libs)
-- 
1.8.3.2

